src/Helper/NumberHelper.php

```diff
@@ -63,11 +64,26 @@ public static function removeTrailingDot(string $input): string
 	 * Preprocess user input for further processing
 	 *
 	 * @param string $input
+	 * @param string[] $decimalPointSigns Default is a dot "." only
+	 * @param string[] $thousandsSeparators
 	 * @return string
 	 */
-	public static function preprocessInput(string $input): string
+	public static function preprocessInput(string $input, array $decimalPointSigns = [], array $thousandsSeparators = []): string
 	{
+		$mergedSeparators = array_merge($decimalPointSigns, $thousandsSeparators);
```

Comment:
Why you must merge it? Can you use design pattern with one parameter with merged arrays?

Reply:
I have simplified the parameter check, however I don't know how to remove the merge, because one is for decimal point and second one is for thousands separator. Which design pattern do you have on mind?